lib/capybara/queries/text_query.rb

```diff
@@ -31,6 +32,16 @@ def failure_message
           message << " but found #{@count} #{Capybara::Helpers.declension('time', 'times', @count)}"
         end
         message << " in #{@actual_text.inspect}"
+
+        if @node and visible?
```

Comment:
well saying a certain number was found in invisible text isn't really right in all cases

    <div>hello he<span>llo he</span>llo</div>

if spans are hidden is it really right to say 1 was found in the invisible text? I think a better way would be to use the total and say something like "However, it was found 3 times when invisible text is included"
If spans are hidden then visible will find 2 instances of 'hello'